README.md

```diff
@@ -1,15 +1,126 @@
-# Holochain Command Line
+# Holochain Command Line Tools
 
-Holochain is being rewritten in Rust as a set of crates that make up the Holochain Library. This means we need a container app to load Holochain and use its features. Using the cmd line wrapper [CLAP](https://github.com/kbknapp/clap-rs ) it was easy to parse the command line and pass those values to Holochain.
-As the library progresses we will add more exposed functions to the command line.
+`holochain-cmd` is a set of tools for building and running [Holochain](https://holochain.org) applications from the command line. The tools are written in Rust, and delivered as binary executables.
+
+## How To Install
+
+Prerequisite: [Rust](https://www.rust-lang.org/en-US/install.html) must be installed on your computer
+
+To install the Holochain command line , run the following commands in a terminal
+```shell
+$ git clone https://github.com/holochain/holochain-cmd.git
+$ cd holochain-cmd
+$ cargo install -f --path .
+```
+
+The command line tools are now available in your command line using the `hcdev` command.
+Run `hcdev -V` to confirm.
+Run `hcdev help` for help.
+
+## Available Commands
+
+`(u)` means the command is as-yet unimplemented.
+
+| Command   | Use                                                                 |
+|-----------|---------------------------------------------------------------------|
+| agent (u) | Starts a Holochain node as an agent                                 |
+| generate  | Generates a new zome and scaffolds the given capabilities           |
+| init      | Initializes a new Holochain app at the given directory              |
+| package   | Builds the current Holochain app into a `.hcpkg` file               |
+| unpack    | Unpacks a Holochain bundle into it's original file system structure |
+| web (u)   | Starts a web server for the current Holochain app                   |
+
+## How To Get Started Building An App
+
+In your terminal, change directories to one where you wish to initialize a new Holochain app.
+Run the following, replacing `your_app_name` with your actual app name:
+```shell
+$ hcdev generate your_app_name
```

Comment:
I believe that would be `init`, not `generate`, right @sphinxc0re?

Reply:
Yes, you're right! oops